scripts/deploy

```diff
@@ -104,21 +102,23 @@ if ! $mock_ctrl; then
   sed -i 's/DISABLE_HW = True/DISABLE_HW = False/' ../amplipi/rt.py
 fi
 
-if $bump; then
-  # Bump version so we can keep track of chanqges
-  poetry version prerelease
+if $dev; then
+  old_version=$(poetry version -s)
+  developer_version="$(git describe --always --dirty)-$(git rev-parse --abbrev-ref HEAD)"
+  # TODO: write extra git info to the pyproject.toml file since poetry complains about non-standard versions
+  #poetry version $developer_version
 fi
 # package release
 poetry build
 
+if $dev; then
```

Comment:
What's going on here?

Reply:
Removed the weird stuff. Added new issue for integrating support for displaying git info on the app

Reply:
#224